File: qlty-cli/tests/lang/vbnet/basic.in/Returns.vb
Imports System

Public Class Returns
    Public Sub F0()
    End Sub

    Public Sub F1()
        Return
    End Sub

    Public Sub F2()
        If True Then
            Return
        Else
            Return
        End If
    End Sub

    Public Sub F3()
        If True Then
            Return
        ElseIf True Then
            Return
        Else
            Return
        End If
    End Sub

    Public Sub F4()
        If True Then
            Return
        ElseIf True Then
            Return
        ElseIf True Then
            Return
        Else
            Return
        End If
    End Sub
End Class

File: qlty-cli/tests/lang/scala/basic.in/Returns.scala
class Returns {
  def manyReturns(a: Int): Int = {
    if (a < 0) return -1
    if (a == 0) return 0
    if (a < 10) return 1
    if (a < 100) return 2
    if (a < 1000) return 3
    if (a < 10000) return 4
    return 5
  }
}
